client/src/session.rs

```diff
@@ -567,10 +567,18 @@ impl SessionLayer {
         false
     }
 
+    pub async fn session(&self, node_id: NodeId) -> Result<RoutingSender, SessionError> {
+        self.session_with_connection_methods(node_id, vec![]).await
+    }
+
     /// Returns `RoutingSender` which can be used to send packets to desired Node.
     /// Creates session with Node if necessary. Function will choose the most optimal
     /// route to destination.
-    pub async fn session(&self, node_id: NodeId) -> Result<RoutingSender, SessionError> {
+    pub async fn session_with_connection_methods(
```

Comment:
This name is exactly the opposite of what it should be: `session_without_connection_methods`